src/screens/LoginPage/LoginPage.tsx

```
@@ -318,7 +318,10 @@ const loginPage = (): JSX.Element => {
     }
   };
 
-  const loginLink = async (e: ChangeEvent<HTMLFormElement>): Promise<void> => {
+  const loginLink = async (
+    e: ChangeEvent<HTMLFormElement>,
+    isAdminRoute: boolean = false,
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)